.prettierrc.js

```diff
@@ -1,9 +1,9 @@
 module.exports = {
-    tabWidth: 4,
```

Comment:
It looks like your JS/TS files doesn't use the following config (including this file). Have you run `yarn prettier`?